src/routes/forms/submission/new/router.ts

```diff
@@ -1,13 +1,11 @@
-import { Request, Response, Router } from "express";
+import { type Request, type Response, Router } from "express";
 
-const router = Router({
+export const newRouter = Router({
```

Comment:
I think the name `newSubmissionRouter` would be more accurate.

Reply:
You are right it feels weird. I guess I just wanted to be consistent with the other ones that I renamed. I should try to find another way of naming them then. Will think about it!